src/main/java/mate/academy/spring/MainApp.java

```diff
@@ -37,15 +64,74 @@ public static void main(String[] args) throws SQLException {
         }
 
         BookService bookService = context.getBean(BookService.class);
-        bookService.add(new Book("Shildt", 2015, new BigDecimal(900)));
-        bookService.add(new Book("Ekkel", 2014, new BigDecimal(1200)));
 
+        // Add Books
+        bookService.add(book1);
+        bookService.add(book2);
+        bookService.add(book3);
+
+        // Get Books
         List<Book> books = bookService.listBooks();
         for (Book book : books) {
-            System.out.println(book.getId());
-            System.out.println(book.getPrice());
-            System.out.println(book.getTitle());
+            System.out.println("Id = " + book.getBookId());
+            System.out.println("Title = " + book.getTitle());
+            System.out.println("Year = " + book.getYear());
+            System.out.println("Price = " + book.getPrice());
+            System.out.println();
         }
+
+        List<Book> bookByTitle = bookService.findByTitle("PIU PIU");
+        for (Book book : bookByTitle) {
+            System.out.println("Id = " + book.getBookId());
+            System.out.println("Title = " + book.getTitle());
+            System.out.println("Year = " + book.getYear());
+            System.out.println("Price = " + book.getPrice());
+            System.out.println();
+        }
+
+        AuthorService authorService = context.getBean(AuthorService.class);
+
+        // Add Authors
+        authorService.add(author1);
+        authorService.add(author2);
+
+        // Get Authors
+        List<Author> authors = authorService.listAuthors();
+        for (Author author : authors) {
+            System.out.println("Id = " + author.getId());
+            System.out.println("Title = " + author.getName());
+            System.out.println("Year = " + author.getSurname());
+            System.out.println();
+        }
+
+        List<Author> authorByName = authorService.findByName("Gorik");
+        for (Author author : authorByName) {
+            System.out.println("Id = " + author.getId());
+            System.out.println("Title = " + author.getName());
+            System.out.println("Year = " + author.getSurname());
+            System.out.println();
+        }
+
+        RentService rentService = context.getBean(RentService.class);
+
+        LocalDate localDate = LocalDate.now();
+        Rent rent1 = new Rent(localDate, user1, book1, true);
```

Comment:
@AlexKukhtinDev true - має бути по дефолту
```suggestion
        Rent rent1 = new Rent(localDate, user1, book1);
```

Reply:
ok